Why: Install init_wslua.h so wiregasm can (re)load Lua plugins.

diff --git a/epan/wslua/CMakeLists.txt b/epan/wslua/CMakeLists.txt
index 8e724cecf8771ff8d688e80f4f15c4487c590daa..a63ce15bdfe4230d48629988e3e0920b4be55728 100644
--- a/epan/wslua/CMakeLists.txt
+++ b/epan/wslua/CMakeLists.txt
@@ -9,6 +9,14 @@
 
 add_subdirectory(lrexlib)
 
+set(WSLUA_PUBLIC_HEADERS
+	init_wslua.h
+)
+
+install(FILES ${WSLUA_PUBLIC_HEADERS}
+	DESTINATION "${PROJECT_INSTALL_INCLUDEDIR}/epan/wslua"
+)
+
 set(WSLUA_MODULES
 	${CMAKE_CURRENT_SOURCE_DIR}/lua_bitop.c
 	${CMAKE_CURRENT_SOURCE_DIR}/wslua_address.c
